### PR TITLE
Feature/parsed logs

### PR DESCRIPTION
**Pull request by ShauliSolomovich**

### CODE REVIEW

File: examples/helm/pbcopy [hunk 1]
```diff
@@ -0,0 +1 @@
+[ -f ~/.fzf.zsh ] && source ~/.fzf.zsh
```
Comment (oded-dd) [inline examples/helm/pbcopy]: Why is it here?

File: examples/helm/templates/configmap.yaml [hunk 2]
```diff
@@ -11,182 +11,21 @@ metadata:
     heritage: {{ .Release.Service }}
 data:
   fluent.conf: |
-    @include conf/*.conf
```
Comment (oded-dd) [inline examples/helm/templates/configmap.yaml]: Dont you want to use all the files under conf/    ?

File: examples/helm/templates/deployment.yaml [hunk 3]
```diff
@@ -0,0 +1,103 @@
+{{- if (eq .Values.kind "Deployment") }}
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "coralogix-fluentd.name" . }}-deployment
```
Comment (oded-dd) [inline examples/helm/templates/deployment.yaml]: Remove the `-deployment`

File: examples/helm/templates/deployment.yaml [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+      annotations:
+{{ toYaml .Values.annotations | indent 8 }}
+    spec:
+      containers:
+      - name: fluentd-coralogix
+        image: {{ .Values.container.image.repository }}:{{ .Values.container.image.tag }}
+        imagePullPolicy: {{ .Values.container.image.pullPolicy }}
+        envFrom:
+        - secretRef:
+           name: {{ template "coralogix-fluentd.name" . }}-account-secrets
```
Comment (oded-dd) [inline examples/helm/templates/deployment.yaml]: Remove the `-account-secrets`

File: examples/helm/values.yaml [hunk 5]
```diff
@@ -1,7 +1,10 @@
 ##  Coralogix Account secrets
-PRIVATE_KEY: XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX
-APP_NAME: $kubernetes.namespace_name
-SUB_SYSTEM: $kubernetes.container_name
+PRIVATE_KEY: 6411e033-3439-d71c-542b-0d45419d6b30
```
Comment (oded-dd) [inline examples/helm/values.yaml]: Remove the private key from the repository

File: examples/helm/config/prod.values.yaml [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+  debug: false
+
+## Pod settings
+container:
+  image:
+    repository: docker.io/coralogixrepo/fluentd-coralogix-image
+    tag: 1.1.4
+    pullPolicy: Always
+  resources:
+    limits:
```
Comment (oded-dd) [inline examples/helm/config/prod.values.yaml]: remove the limits for now it is running on its own instance so we would like to examine its performance

File: examples/helm/config/prod.values.yaml [hunk 7]
```diff
@@ -0,0 +1,82 @@
...
+      cpu: 100m
+      memory: 400Mi
+    requests:
+      cpu: 100m
+      memory: 400Mi
+
+## Environment variables
+env:
+  CORALOGIX_LOG_URL: "https://test.coralogix.com/api/v1/logs"
```
Comment (oded-dd) [inline examples/helm/config/prod.values.yaml]: This is the URL for stage?

File: examples/helm/config/prod.values.yaml [hunk 8]
```diff
@@ -0,0 +1,82 @@
...
+
+## Security settings
+rbac:
+  create: true
+
+## Node settings
+tolerations:
```
Comment (oded-dd) [inline examples/helm/config/prod.values.yaml]: It shouldn't be, as your already configured it below

File: examples/helm/config/stage.values.yaml [hunk 9]
```diff
@@ -0,0 +1,82 @@
+##  Coralogix Account secrets
+PRIVATE_KEY:
```
Comment (oded-dd) [inline examples/helm/config/stage.values.yaml]: This file can be removed. We will create it in the future with Dhall

File: examples/helm/templates/configmap.yaml [hunk 10]
```diff
@@ -11,267 +11,80 @@ metadata:
...
+    #@include conf/kubernetes.conf
```
Comment (oded-dd) [inline examples/helm/templates/configmap.yaml]: Dont you want to use all the files under conf/ ?

File: examples/helm/values.yaml [hunk 11]
```diff
@@ -1,11 +1,14 @@
 ##  Coralogix Account secrets
```
Comment (oded-dd) [inline examples/helm/values.yaml]: This file can also be delete as we later on will re-write it in Dhall

File: examples/helm/config/prod.values.yaml (hunk 7 above)
Comment (ShauliSolomovich) [reply examples/helm/config/prod.values.yaml]: yes

File: examples/helm/templates/configmap.yaml (hunk 10 above)
Comment (ShauliSolomovich) [reply examples/helm/templates/configmap.yaml]: yes, on the image the files from the configMap go to this path, we also need to change it

**Pull request merged**